playbooks/roles/plugins/foreman_openscap/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+- name: 'Install openscap plugin'
+  include_role:
+    name: plugins/plugin_installer
+  vars:
+    plugin_installer_options: >
+      --enable-foreman-plugin-openscap
+      --enable-foreman-proxy-plugin-openscap
+      {{ foreman_openscap_installer_options }}
+  tags:
+    - installation
+
+- name: 'Install foreman openscap hammer plugin'
+  yum:
```

Comment:
It doesn't install the hammer plugin for me. It might be an installer issue but I wouldn't block this PR until we fix it.